docs/ERC-725.md

```diff
@@ -3,7 +3,7 @@ eip: 725
 title: General key-value store and execution standard
 author: Fabian Vogelsteller (@frozeman), Tyler Yasaka (@tyleryasaka)
 discussions-to: https://github.com/ethereum/EIPs/issues/725
-status: Draft
+status: Final
```

Comment:
Maybe we should not put the EIP as Final, but leave it as "Draft" or "Review" instead, as:
- we have recently made quite a lot of recurring BREAKING CHANGES.
- we might not be aware of potential issues coming yet with the interface changes we made.
- it is still quite early stage

Reply:
Good point. @frozeman Thoughts ?

Reply:
Review